RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -793,6 +1304,61 @@ TauVars AntiElectronIDMVA6<TauType, ElectronType>::getTauVarsTypeSpecific(const
   return tauVars;
 }
 
+// reco::GsfElectron
+template <class TauType, class ElectronType>
+void AntiElectronIDMVA6<TauType, ElectronType>::getElecVarsHGCalTypeSpecific(
+    const reco::GsfElectronRef& theEleRef, antiElecIDMVA6_blocks::ElecVars& elecVars) {
+  //MB: Assumed that presence of one of the HGCal EleID variables guarantee presence of all
+  if (!(electronIds_.find("hgcElectronID:sigmaUU") != electronIds_.end() &&
+        electronIds_.at("hgcElectronID:sigmaUU").isValid()))
+    return;
```

Comment:
this is independent of the input arguments.
Isn't there a better place to check this consistency? (e.g. where `electronIds_` are filled)

Reply:
This somehow does the same as similar check for pat::Electron for presence of userFloat. I am not sure it can be done in a significantly nicer way. Do you have some particular idea in mind?

Reply:
not really, I didn't check the call structure to see if one can know if `electronIds_` are done being filled where they are (constructor or some "set" method?) and just do the check there, could be a bool flag set